### PR TITLE
minor UI changes and addition of button on server show page

### PR DESCRIPTION
**Pull request by Evana13G**

### CODE REVIEW

File: app/routes/servers/show.coffee [hunk 1]
```diff
@@ -7,6 +7,10 @@ ServersShowRoute = Ember.Route.extend({
   actions:
     executeTests: ->
       @transitionTo('servers.test', @get('currentModel.server'))
+    
+    testRunsPage: ->
+      @transitionTo('test_runs.show', @get('currentModel.testRun'))
```
Comment (noranda) [inline app/routes/servers/show.coffee]: Did you want the Test Results button to automatically run all tests?

File: app/templates/components/server-details.hbs [hunk 2]
```diff
@@ -1,6 +1,8 @@
 <div class="row server-details">
   <div class="col-md-3 col-md-offset-2 server-icon">
-    <i class="fa fa-server"></i>
+    {{#link-to 'servers.show' server}}
+      <i class="fa fa-server"></i>
+    {{/link-to}}
```
Comment (noranda) [inline app/templates/components/server-details.hbs]: We're getting some errors when going to the `servers.show` page from here.

Imgur

File: app/routes/servers/show.coffee (hunk 1 above)
Comment (Evana13G) [inline app/routes/servers/show.coffee]: no. But Schreiber mentioned that he has to fix something first and to leave the button there.

File: app/templates/components/server-details.hbs (hunk 2 above)
Comment (Evana13G) [inline app/templates/components/server-details.hbs]: I cannot see that image

**Pull request closed without merging**